packages/lib-react-components/src/Media/components/ThumbnailImage/ThumbnailImage.js

```diff
@@ -85,7 +84,5 @@ ThumbnailImage.propTypes = {
 }
 
 ThumbnailImage.defaultProps = {
```

Comment:
This should be removed and replaced with default function args.

Reply:
Good point! I'll add default function args instead.